cerbero/packages/osx/packager.py

```diff
@@ -234,26 +236,32 @@ def pack(self, output_dir, devel=False, force=False, keep_temp=False, split=True
         PackagerBase.pack(self, output_dir, devel, force, keep_temp, split)
 
         self._prepare_pack()
-
-        if isinstance(self.package, MetaPackage):
+        paths = []
+        try:
+            # Create osx-bundle package (self.package is ALWAYS a MetaPackage)
```

Comment:
The documentation of the class already says this:
"Creates an osx package from a L{cerbero.package.package.MetaPackage}"

Reply:
yes but the code before was checking is isinstance(self.package, MetaPackage), so I think it's not a bad idea to insist here ;)